### PR TITLE
Feature/sentry integration

### PR DESCRIPTION
**Pull request by Johanndutoit**
Added raven and added the newer implementation details from the doc over at https://github.com/getsentry/raven-node#express on how to install for Express.

### CODE REVIEW

**Comment by Johanndutoit** [comment]
@smn it looks for the SENTRY_DSN environment variable

**Comment by Johanndutoit** [comment]
@jonathanendersby @BarryBotha thoughts ?

File: goddard/httpd.coffee [hunk 1]
```diff
@@ -3,11 +3,15 @@
...
 bodyParser 		= require('body-parser')
 session 		= require('express-session')
+raven 			= require('raven');
 RedisStore 		= require('connect-redis')(session);
 
 # create the instance to setup and use
 app = express()
 
+# Should be the first item listed
+app.use(raven.middleware.express.requestHandler(process.env.SENTRY_DSN or ''))
```
Comment (smn) [inline goddard/httpd.coffee]: Perhaps only call `app.use` if the `SENTRY_DSN` is defined? Not sure if it trips up over an empty string as a DNS.

Comment (Johanndutoit) [inline goddard/httpd.coffee]: Good catch, updated to not create the middleware if variable was not given

**Comment by smn** [comment]
:+1: from me.

**Pull request merged**